misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/PickQuantityPrompt.jsx

```diff
@@ -29,14 +29,15 @@ class PickQuantityPrompt extends PureComponent {
 
   render() {
     const { qtyTarget, qtyCaption, onCloseDialog } = this.props;
+    const target = qtyTarget > 0 ? qtyTarget : 0;
```

Comment:
instead of `target` you could 
 * name it `qtyTargetNorm` (Norm from normalized) 
* or `let qtyTarget = Math.min(this.props.qtyTarget, 0);`